GooglePlacesAutocomplete.js

```diff
@@ -160,7 +160,22 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
   const [listLoaderDisplayed, setListLoaderDisplayed] = useState(false);
 
   const inputRef = useRef();
-  const [sessionToken, setSessionToken] = useState(uuidv4());
+
+  const randomGenerator = useCallback(() => {
+    const randomByteArray = [];
+    for (let i = 0; i < 16; i++) {
+      const rand = Math.floor(Math.random() * 256);
+      randomByteArray.push(rand);
+    }
+    return randomByteArray;
+  });
+
+  const generateSessionToken = useCallback(() => {
+    return uuidv4({ rng: randomGenerator });
```

Comment:
As we had an issue concerning usage of crypto.getRandomValues on iOS Native devices, we can use a random generator for session token generation which I think is good enough for the given scenario. This change works well on iOS devices.

Reply:
I can't think of any reason these need to be cryptographically secure, can you? If not, this seems fine to me 👍

Reply:
> I can't think of any reason these need to be cryptographically secure, can you? If not, this seems fine to me

Yeah. I don't see any valid reason either to make it so secure.